pkg/chart/chart.go

```diff
@@ -284,13 +284,11 @@ func (t *Testing) LintChart(chart string, valuesFiles []string) TestResult {
 		result.Error = err
 		return result
 	}
-	if err := t.linter.YamlLint(chartYaml, t.config.LintConf); err != nil {
-		result.Error = err
-		return result
-	}
-	if err := t.linter.YamlLint(valuesYaml, t.config.LintConf); err != nil {
-		result.Error = err
-		return result
+	for _, yamlFile := range append(valuesFiles, valuesYaml, chartYaml) {
```

Comment:
I'd suggest you don't append stuff to the method parameter. Please create a new slice and append everything. Also, appending `chartYaml` to `valuesFiles` is not ideal either.